packages/mobile/src/exchange/ExchangeHomeScreen.tsx

```diff
@@ -1,5 +1,5 @@
 import ScrollContainer from '@celo/react-components/components/ScrollContainer'
-import SectionHead from '@celo/react-components/components/SectionHead'
```

Comment:
Can you make an issue to remove the old one and rename new to just `SectionHead`

Reply:
Opened #872